packages/react-urql/examples/1-getting-started/public/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="charset" value="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>urql Example: 1. Getting Started</title>
+
+    <script>
+      navigator.serviceWorker.getRegistrations().then(registrations => {
+        // Unregister all possible CodeSandbox or other service-workers
+        for (const ctx of registrations) {
+          if (ctx.active && !ctx.active.scriptURL.endsWith("sw.js")) {
+            ctx.unregister();
+          }
+        }
+
+        // Register the mock API service-worker then close all registrations
+        navigator.serviceWorker.register("/sw.js", { scope: "./" });
+        navigator.serviceWorker.register = () => Promise.resolve({});
```

Comment:
This may not be needed anymore, but basically all this is a magic incantation... 😆  see: https://github.com/FormidableLabs/urql/pull/624#discussion_r396757417